app/controllers/recipes_controller.rb

```diff
@@ -7,7 +7,9 @@ def index
   end
 
   # GET /recipes/1 or /recipes/1.json
-  def show; end
+  def show
+    @recipe_food = RecipeFood.all
+  end
```

Comment:
:red_circle:  I think this show method should return a recipe item. Then You can lately in your `show.html.erb` template access all the recipe_foods for this recipe item using the `@recipe.recipe_foods`
I think this method should look like this :arrow_down: 
```
  def show
    @recipe = Recipe.find(params[:id])
  end
```